component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -1588,13 +1590,14 @@ protected boolean retryAuthenticationEnabled() {
      * @throws AuthenticationFailedException : In occasions of failing sending the email to the user.
      */
     protected void triggerEvent(String userName, String tenantDomain, String userStoreDomainName,
-                                String notificationEvent, String otpCode) throws AuthenticationFailedException {
```

Comment:
This is an API Change. better to deprecate this call, and have a new private method.

Reply:
This existing API was introduced in last WUM update and that is why we thought to change it. If that is a problem, +1 to add another method.